src/executables/unload_schema.c

```diff
@@ -773,7 +780,15 @@ export_serial (print_output & output_ctx)
 
       output_ctx ("call [find_user]('%s') on class [db_user] to [auser];\n",
 		  db_get_string (&values[SERIAL_OWNER_NAME]));
-      output_ctx ("create serial %s%s%s\n", PRINT_IDENTIFIER (db_get_string (&values[SERIAL_NAME])));
+
+      serial_name_copy = strdup (db_get_string (&values[SERIAL_FULL_NAME]));
+      owner_name_p = strtok_r (serial_name_copy, ".", &save_token);
+      serial_name_p = strtok_r (NULL, ".", &save_token);
```

Comment:
Consider using strchr()

Reply:
I will use this method while changing to a macro.